code/game/turfs/open.dm

```diff
@@ -4,6 +4,114 @@
 	var/wet = 0
 	var/wet_time = 0 // Time in seconds that this floor will be wet for.
 	var/image/wet_overlay = null
+	var/turf_gravity_overrides_area = FALSE	//USE AREAS. NOT THIS. THIS IS ONLY FOR VERY LOCALIZED EFFECTS.
```

Comment:
The object is just for tracking vars so they don't change unnecessarily to save on processing time usage . Trade off is the obj cars existing.